client-spark/common/src/main/java/org/apache/spark/shuffle/RssClientConfig.java

```diff
@@ -79,4 +79,11 @@
   // has huge amount blockIds, eg, 10B.
   public static String RSS_CLIENT_BITMAP_SPLIT_NUM = "spark.rss.client.bitmap.splitNum";
   public static int RSS_CLIENT_BITMAP_SPLIT_NUM_DEFAULT_VALUE = 1;
+  public static String RSS_ACCESS_ID = "spark.rss.access.id";
+  public static String RSS_ACCESS_TIMEOUT_MS = "spark.rss.access.timeout.ms";
+  public static int RSS_ACCESS_TIMEOUT_MS_DEFAULT_VALUE = 10000;
+  public static String RSS_USE_RSS_SHUFFLE_MANAGER = "spark.rss.useRssShuffleManager";
+  public static boolean RSS_USE_RSS_SHUFFLE_MANAGER_DEFAULT_VALUE = false;
+  public static String RSS_SORT_SHUFFLE_MANAGER_IMPL = "spark.rss.sort.shuffle.manager.impl";
```

Comment:
The parameter is unnecessary, make org.apache.spark.shuffle.sort.SortShuffleManager as a constant.

Reply:
This param gives users the ability to specify their own shuffle manager.

Reply:
It's unnecessary

Reply:
removed